plugins/orchestrator-backend/src/service/WorkflowService.ts

```diff
@@ -81,12 +102,17 @@ export class WorkflowService {
   }
 
   async saveOpenApi(): Promise<void> {
-    const path = this.resolveRosourcePath(actions_open_api_file_path);
+    const path = this.resolveResourcePath(actions_open_api_file_path);
     const openApi = await this.openApiService.generateOpenApi();
     if (!openApi) {
       return;
     }
     await this.saveFile(path, openApi);
+
+    await this.githubService.push(
```

Comment:
One future improvement in this area could be pushing all related changes in one single commit.
For instance: a workflow and its spec files.

Reply:
yep I agree, and another thing I'll add is the option to enable/disable the push using a config param